data/mp/eda_mp_trj.py

```diff
@@ -64,19 +64,17 @@
 # %% extract extXYZ files from zipped directory without unpacking the whole archive
 # takes ~8 mins on M2 Max
 # takes ~5 mins on M3 Max
-atoms_list = ase_atoms_from_zip(
-    zip_path, file_check=lambda name: name.startswith("mptrj-gga-ggapu/mp-")
-)
+atoms_list = ase_atoms_from_zip(zip_path)
 
 mp_trj_atoms: dict[str, list[ase.Atoms]] = defaultdict(list)
 for atoms in atoms_list:
-    mp_id = atoms.info.get("mp_id", "no-id")
-    assert mp_id.startswith("mp-")
+    mp_id = atoms.info.get(Key.mat_id, "no-id")
```

Comment:
This doesn't work? Key.mat_id - `mat_id = "material_id", "Material ID"` the key in the json/dict is "mp-id"

Reply:
actually I renamed in the updated file didn't I.

Reply:
it worked for me just now. try `Key.mat_id == "material_id"`. not `True`?

Reply:
in the old file it was "mp-id" but had been bulk changed to Key.mat_id. Because of that when I remade I renamed it to Key.mat_id and then got my wires crossed when I saw this as it was broken for old but works for new file